.github/e2e-tests-olm/action.yaml

```diff
@@ -33,10 +33,14 @@ runs:
         wait: 300s
         config: ./hack/kind/config.yaml
 
+    - name: Install required tools
+      shell: bash
+      run: |
+        make operator-sdk promq
+
     - name: Install OLM
       shell: bash
       run: |
-        make operator-sdk
         ./tmp/bin/operator-sdk olm install
```

Comment:
We can consider inlining this command now that there's only one thing to execute